scripts/demo-fruits.sh

```diff
@@ -10,51 +10,24 @@ source ${SCRIPTS_DIR}/play-demo.sh
 ####################################
 VM_IP=${VM_IP:=127.0.0.1}
 NAMESPACE=app
-REGISTRY=localhost:5000
-IMAGE_VERSION=latest
+FRUITS_CHART_NAME=fruits-app
+IMAGE=quay.io/halkyonio/atomic-fruits:latest
 INGRESS_HOST=atomic-fruits.${VM_IP}.nip.io
-QUARKUS_APP_PATH="$HOME/atomic-fruits-service"
-GITHUB_REPO=https://github.com/aureamunoz/atomic-fruits-service
-
-# Database credential
-DB_NAMESPACE=db
-USERNAME=healthy
-PASSWORD=healthy
-TYPE=postgresql
 
 # Parameters to play the demo
 TYPE_SPEED=${TYPE_SPEED:=200}
 NO_WAIT=true
 
-if [[ ! -d "${QUARKUS_APP_PATH}" ]]; then
-  git clone ${GITHUB_REPO} ${QUARKUS_APP_PATH}
-fi
-
 if [[ "$1" == "clean" ]]; then
-  pe "helm uninstall postgresql -n ${DB_NAMESPACE} || true"
-  pe "k delete -f ${QUARKUS_APP_PATH}/target/kubernetes/kubernetes.yml -n ${NAMESPACE} || true"
-  pe "k delete ns ${NAMESPACE} || true"
-  pe "k delete ns ${DB_NAMESPACE} || true"
+  pe "helm uninstall ${FRUITS_CHART_NAME} -n ${NAMESPACE} || true"
```

Comment:
What about removing the Helm database dependency if this property is enabled - https://github.com/halkyonio/atomic-fruits-service/blob/main/src/main/resources/application.properties#L31 ?

Reply:
I don't understand your question: The Helm database will be removed when removing the fruits chart if it was enabled.

Reply:
I still don't follow you. With this command, you're going to delete the fruits helm chart (regardless the database was installed `db.enabled=true` or not). If the database was installed which is the default, it will be deleted as well.

Reply:
`If the database was installed which is the default, it will be deleted as well.` Is it because it is defined as helm dependency ?